009_web_L370/009_web_L370_unsafe.js

```
@@ -0,0 +1,598 @@
+        query = AnnotationModel.find({
+          'uri': { $regex: exd, $options: "i" }
+        });
+      }
+      break;
+    case 'dashboard':
+      query = AnnotationModel.find();
+      query.where('uri').regex(re);
+      break;
+    case 'public':
+      query = AnnotationModel.find();
+      var pattern = new RegExp("\/public\/(.+)$", 'i');
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)